### ISSUE TITLE
Add tl_content.singleSRC support

### ISSUE THREAD

**Issue by creffect** [inspiredminds/contao-file-usage#5]
Thank you for this great extension.

In my case, I needed to write a custom FileUsageProvider to check the  tl_content.singleSRC field?

Is this an error on my side or intended usage? (because the image-Content-Element is Contao Core and is using the fileTree, I thought it would be supported by default of this extension).

If not, here is a simple Provider to integrate: 

```php
public function find(): ResultsCollection {
    $collection = new ResultsCollection();
    $contents = ContentModel::findAll([
        'column' => ['singleSRC IS NOT NULL'],
        'value' => [],
    ]);
    foreach ($contents as $content) {
        $uuid = $content->singleSRC;
        if (Validator::isBinaryUuid($uuid)) {
            $uuid = StringUtil::binToUuid($uuid);
            $collection->addResult($uuid, new DatabaseReferenceResult("tl_content", "singleSRC", $content->id));
        }
    }
    return $collection;
}
```

**Comment by fritzmg** [comment]
Hm, no that would be a bug. The integrated `DatabaseProvider` should cover that.

Which Contao Version?

**Comment by creffect** [comment]
Contao 5.3.4. 

This are the composer dependencies:

"require": {
        "contao/conflicts": "@dev",
        "contao/listing-bundle": "5.3.4",
        "contao/manager-bundle": "5.3.4",
        "madeyourday/contao-rocksolid-custom-elements": "v2.4.10",
        "madeyourday/contao-rocksolid-frontend-helper": "v2.2.6",
        "twbs/bootstrap": "v5.2.3",
        "inspiredminds/contao-file-usage": "2.2.2"
    },

**Comment by fritzmg** [comment]
I could not reproduce the problem. Here is what I tried in Contao 5.3.x:

1. Created an `image` content element and selected an image.
2. Executed `contao_file_usage:warmup`.
3. Went to _Files_ and clicked on the ![link](https://github.com/inspiredminds/contao-file-usage/assets/4970961/cf15dcac-5401-46fc-a929-bfd5ac2b12b5) icon.

The content element was listed.

**Comment by creffect** [comment]
Thank you, I rechecked and it works. Probably it was a misunderstanding, I didn't know that I had to run "contao_file_usage:warmup" manually after clearing the cache and warming it up.

My workflow was
1) clear cache `cache:clear`
2) warmup cache `cache:warmup`

Now all files seem unused. I need to run `contao_file_usage:warmup` afterwards. Maybe it would be possible to include `contao_file_usage:warmup` into the `cache:warmup` command, for a more intuitive usage. 

I will include it in my bash scripts and removed my manual FileUsage-Provider.

**Comment by fritzmg** [comment]
> Maybe it would be possible to include `contao_file_usage:warmup` into the `cache:warmup` command, for a more intuitive usage.

That is inadvisable as this will take far too long, depending on your Contao instance.

Arguably the fileusage cache should may be not reside in the environment's cache directory, so that it does not get cleared during `cache:warmup`.

You could override the service definition in your setup this way:

```yaml
# config/services.yaml
services:
    contao_file_usage.file_usage_cache:
        class: Symfony\Component\Cache\Adapter\FilesystemAdapter
        public: true
        autoconfigure: true
        arguments: ['fileusage', 0, '%kernel.project_dir%/var']
```

(or change the argument via a Compiler Pass)

This will put the fileusage cache to `var/fileusage/` rather than `var/cache/<env>/fileusage/`.

**Comment by fritzmg** [comment]
I have released `2.3.0` now where the cache now resides in `var/fileusage/`.
